docs/developer/release-process.md

```diff
@@ -109,6 +109,7 @@ yarn upgrade
 ```bash
 go mod tidy
 go list -u -f '{{if (and (not (or .Main .Indirect)) .Update)}}{{.Path}}: {{.Version}} -> {{.Update.Version}}{{end}}' -m all 2> /dev/null
+go get -u ./...
```

Comment:
This conflicts with the next line, I think? Assuming the above automatically updates all go dependencies? Perhaps if it works as you've documented (this time), include it in the following sentence as an option if there are no issues. See what you think.

Reply:
I misread it when I wrote the guide, the `go list -u...` is just for _To view available minor and patch upgrades only for the direct dependencies_

And:

`go get -u ./... `to use the latest minor or patch releases (and add -t to also upgrade test dependencies)
`go get -u=patch ./... `to use the latest patch releases (and add -t to also upgrade test dependencies)

So let me reword the text to make it more explicit. Thanks for pointing it out.